tests/main/security-dev-input-event/task.yaml

```diff
@@ -0,0 +1,68 @@
+summary: Ensure that /dev/input/event* is denied by default.
+
+details: |
+    The default policy disallows access to /dev/input/event*.
+
+    The joystick interface disallows access to /dev/input/event* for
+    non-joysticks.
+
+    The test checks the snap is not able to access /dev/input/event* with or
+    without the joystick interface connected. We do this since the
+    /dev/input/event* devices are sensitive and because the joystick interface
+    adds a /dev/input/event* AppArmor glob rule that relies entirely on the
+    device cgroup for enforcement.
+
+prepare: |
+    echo "Given the test-snapd-event snap is installed"
+    snap install test-snapd-event
+
+restore: |
+    rm -f call.error
+    snap remove test-snapd-event
```

Comment:
snaps are removed automatically on reset script, it is not needed. The idea is thta the prepare/restore scripts will make sure any test starts with the same snapd state.

Reply:
Done